testing/src/scenario/_ops_main_mock.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python3
```

Comment:
This is actually a refactor of `ops_main_mock.py` and not a completely new file, right?

Reply:
Yes. The first commit only renames the files and does nothing else so that it's easier to see, if you want to look that way. I'm unsure why the diff shows as changes for runtime, but not for ops_main_mock - maybe "too much" has changed?